### PR TITLE
[TF 2.0 API Docs] Added Documentation to tf.keras.activations.selu

### LINKED ISSUE #27657: [TF 2.0 API Docs] tf.keras.activations.selu
**System information**
- TensorFlow version: 2.0 alpha
- Doc Link:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/selu
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py

**- Links**
Should format the referenced paper - get rid of "-" and add the authors/year of publishing:
`"Self-Normalizing Neural Networks" (Klambauer et al, 2017)"`

**- Definition**
The current definition does not specify the fixed values for `alpha` and `scale` constants. It also assumes knowledge of the ELU activation function. We should also include a link (i.e. https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/elu).

Proposed modified definition:

```
The Scaled Exponential Linear Unit (SELU) activation function is:

`scale` * `x` if `x > 0` and `scale * alpha * (exp(x)-1)` if `x < 0`

where `alpha` and `scale` are pre-defined constants (`alpha = 1.6732632423543772848170429916717` and `scale = 1.0507009873554804934193349852946`.
The SELU activation function multiplies  `scale` > 1 with the `[elu](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/elu)` (Exponential Linear Unit (ELU)) to ensure a slope larger than one for positive net inputs. 
```

Followed by what is already in the docs with the formatted `lecun_normal` initialization bit and a link to it: https://www.tensorflow.org/api_docs/python/tf/keras/initializers/lecun_normal:

```
...The values of alpha and scale are chosen so that the mean and variance of the inputs are preserved between two consecutive layers as long as the weights are initialized correctly (see `[lecun_normal` initialization](https://www.tensorflow.org/api_docs/python/tf/keras/initializers/lecun_normal)) and the number of inputs is "large enough" (see references for more information).
```

**- Examples**
Can add a modified example fro the Intro to CNNs tutorials (use `selu` instead of `relu`)

```
model = models.Sequential()
model.add(layers.Conv2D(32, (3, 3), activation='selu', input_shape=(28, 28, 1)))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='selu'))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='selu'))
```

**- Returns**
Can modify to include the word `function`:
```
The scaled exponential unit activation function`: `scale * elu(x, alpha)`.
``` 
and format markdown for plain text.

**- Raises**
Not defined.

**- Visuals**
Should be added, similar to: https://cdn-images-1.medium.com/max/1600/1*WyQS-lnoemRA3_FpRL7r5w.png

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
Yes

### PR DESCRIPTION
**Pull request by bharatr21**
Fix #27657 by adding required information to tf.keras.activations.selu [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py) and added examples of FNN instead of ConvNets because SELU finds better use there. Also added an image of the function, hoping the markdown renders properly.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -103,11 +128,14 @@ def selu(x):
...
-      - To be used together with the initialization "lecun_normal".
-      - To be used together with the dropout variant "AlphaDropout".
+      - To be used together with the initialization "[lecun_normal]
+      (https://www.tensorflow.org/api_docs/python/tf/keras/initializers/lecun_normal)".
+      - To be used together with the dropout variant "[AlphaDropout]
+      (https://www.tensorflow.org/api_docs/python/tf/keras/layers/AlphaDropout)".
 
   References:
-      - [Self-Normalizing Neural Networks](https://arxiv.org/abs/1706.02515)
+  [Self-Normalizing Neural Networks (Klambauer et al, 2017)]
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
      [Self-Normalizing Neural Networks (Klambauer et al, 2017)]
```

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -103,11 +128,14 @@ def selu(x):
...
+  (https://arxiv.org/abs/1706.02515)
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
      (https://arxiv.org/abs/1706.02515)
```

**Review by kyscg — CHANGES_REQUESTED** [review]
Good work @Bharat123rox , especially the addition of links.
 
Just fix these and it'll be good to merge.

**Comment by Bharat123rox** [comment]
@kyscg @rthadur @googlebot any ways to confirm the CLA if all co-authors of the commit have already signed it (Even though individual commits are made at later stages)? Is this level of granularity necessary (for each commit?) just asking out of curiosity

**Review by kyscg — DISMISSED** [review]

**Comment by kyscg** [comment]
I think it is necessary, there is no other way of knowing whether all the co-authors are aware of a proposed merge. It's not that complex either.

**Comment by kyscg** [comment]
@rthadur, gentle ping to review and set CLA's to yes

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -89,12 +89,37 @@ def elu(x, alpha=1.0):
...
+
+  ![](https://cdn-images-1.medium.com/max/1600/1*m0e8lZU_Zrkh4ESfQkY2Pw.png)
+  (Courtesy: Blog on Towards DataScience at
+  https://towardsdatascience.com/selu-make-fnns-great-again-snn-8d61526802a9)
+
+  Example Usage:
+  ```python3
+  n_classes = 10 #10-class problem
+  model = models.Sequential()
+  model.add(Dense(64, activation='selu', input_shape=(28, 28, 1)))
```
Comment (ekerazha) [inline tensorflow/python/keras/activations.py]: Shouldn't you add `kernel_initializer='lecun_normal'` ?

Comment (Bharat123rox) [reply tensorflow/python/keras/activations.py]: Added `kernel_initializer='lecun_normal'`, however note that usually `'glorot_uniform'` is recommended, and `activation='selu'` need not indicate otherwise.

**Comment by Bharat123rox** [comment]
Sorry again @kyscg but as @ekerazha pointed out, the `kernel_initializer='lecun_normal'` has been used. Please re-review

**Comment by rthadur** [comment]
@Bharat123rox thank you for your contribution , please make sure all the contributors have signed CLA

**Comment by Bharat123rox** [comment]
Yes, all contributors have signed the CLA @Bharat123rox and @kyscg and are
willing to merge contributions in source code

File: tensorflow/python/keras/activations.py (hunk 3 above)
Comment (ekerazha) [reply tensorflow/python/keras/activations.py]: @Bharat123rox SELU needs lecun_normal initialization and, when used, AlphaDropout instead of Dropout

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -89,12 +89,38 @@ def elu(x, alpha=1.0):
...
+  (Courtesy: Blog on Towards DataScience at
+  https://towardsdatascience.com/selu-make-fnns-great-again-snn-8d61526802a9)
+
+  Example Usage:
+  ```python3
+  n_classes = 10 #10-class problem
+  model = models.Sequential()
+  model.add(Dense(64, kernel_initializer='lecun_normal', activation='selu',
+  input_shape=(28, 28, 1))))
+  model.add(Dense(32, activation='selu'))
```
Comment (ekerazha) [inline tensorflow/python/keras/activations.py]: Add kernel_initializer='lecun_normal' here too

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -89,12 +89,38 @@ def elu(x, alpha=1.0):
...
+  model.add(Dense(16, activation='selu'))
```
Comment (ekerazha) [inline tensorflow/python/keras/activations.py]: Add kernel_initializer='lecun_normal' here too

**Review by ekerazha — COMMENTED** [review]
I think you should add kernel_initializer='lecun_normal' to all hidden layers

**Comment by Bharat123rox** [comment]
@gbaned @rthadur please change CLAs to yes

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -89,12 +89,38 @@ def elu(x, alpha=1.0):
 def selu(x):
   """Scaled Exponential Linear Unit (SELU).
 
-  SELU is equal to: `scale * elu(x, alpha)`, where alpha and scale
-  are pre-defined constants. The values of `alpha` and `scale` are
+  The Scaled Exponential Linear Unit (SELU) activation function is:
+  `scale * x` if `x > 0` and `scale * alpha * (exp(x) - 1)` if `x < 0`
+  where `alpha` and `scale` are pre-defined constants
+  (`alpha = 1.6732632423543772848170429916717`
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: trim

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -89,12 +89,38 @@ def elu(x, alpha=1.0):
...
+  and `scale = 1.0507009873554804934193349852946`).
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: we really don't need that many digits here.

**Comment by Bharat123rox** [comment]
@rthadur @gbaned please add the `cla:yes` labels and @tanzhenyu please re-review

**Review by tanzhenyu — APPROVED** [review]
